### PR TITLE
test : entry snapshotting yo ass testing

### PR DESCRIPTION
**Pull request by franzsamilo**

### CODE REVIEW

File: jest.config.ts
Comment (cezro) [inline jest.config.ts:1]: but why tho

Comment (franzsamilo) [reply jest.config.ts:1]: does it conflict

**Review by cezro — APPROVED** [review]
wow ok
444139050_2245966309090401_919873382891737983_n

File: jest.config.ts
Comment (cezro) [reply jest.config.ts:1]: nah just why

**Pull request merged**